### DISCUSSION TITLE
Still getting Server.dat is damaged

### DISCUSSION THREAD

**Discussion by danilocolasso** [shekohex/coemu#92] [General]
Hey there, thanks for your repo

I'm still getting the Server.dat is damaged
![image](https://github.com/shekohex/coemu/assets/2896393/734e40b0-5738-4b57-868c-1b02c8fb4a14)

Both auth and game servers are up
![image](https://github.com/shekohex/coemu/assets/2896393/cf40e143-6037-4520-baa2-4a1f96853847)

My Server.dat
![image](https://github.com/shekohex/coemu/assets/2896393/edec47ab-2373-4c08-9540-be370724de76)

What could I am doing wrong?

**Comment by shekohex** [comment]
Yeah that is expected, I mentioned that somewhere in the README, but basically Conquer.exe will reject any local IP address and say it is damaged. So you have two options
1. Use a different remote IP (not a local one, like using a loopback device) see tailscale as an example. 
2. Another way, is to use Conquer Loader V5 from here: https://www.elitepvpers.com/forum/co2-programming/610999-release-conquerloader-v2.html which is the easiest way to get started.

> You may want to add it to your Windows defender allow list / exclude it from scanning, because it sees it as a virus/injector.

Hope that helps.

**Reply by danilocolasso** [reply]
Oh, I saw that but I hadn't understood
Got it, thank you @shekohex
